frontend/src/utils/date.ts

```diff
@@ -38,10 +38,14 @@ export const isDateAfterToday = (date: number | Date) => {
   return isAfter(date, endOfToday())
 }
 
-// Converts UTC time to the same date in local time, ignoring original timezone.
-export const fromUtcToLocalDate = (date?: Date | null) => {
+export const normalizeDateToUtc = (date: Date | null) => {
   if (!date) return date
-  return new Date(date.getUTCFullYear(), date.getUTCMonth(), date.getUTCDate())
+  return new Date(date.valueOf() - date.getTimezoneOffset() * 60 * 1000)
```

Comment:
1. Can use date-fns { addMinutes }
`return addMinutes(date, date.getTimezoneOffset())`

2. This will be correct for most cases but not all. That's because `.getTimezoneOffset()` do not always return a uniform number (Daylight Savings Time). [See](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTimezoneOffset)
What could happen is that we normalize date -> not yet DST -> offset is 300
Then loading data back -> is DST -> offset is 240
Your final result will end up being different. 
Some[ relevant discussions](https://github.com/date-fns/date-fns/issues/1401#issuecomment-621897094) and [other potential solutions](https://stackoverflow.com/questions/948532/how-to-convert-a-date-to-utc)
Though honestly not sure if this is a cause for concern for us. 

[Kinda relevant again HAHAHA](https://gist.github.com/timvisee/fcda9bbdff88d45cc9061606b4b923ca)

Reply:
Ugh. Daylight savings.

Reply:
Hm I guess that's why `fromUtcToLocalDate` was done that way.

Reply:
fixed in f0dbefb0